### ISSUE TITLE
FR: Show the % of savings of compression and deduplication

### ISSUE THREAD

**Issue by Tom-H-L** [borgbase/vorta#1468]
It would be nice to see the percentage saved by compression and deduplication in the backup status at the bottom while backing up and also in the Archives tab in terms of "Original Size", "Size", "Saved %".

**Comment by m3nu** [comment]
Agreed on that. While Borg doesn't have this info, Vorta does calculate the size of the source folder on its own.

Sadly this wouldn't cover the case where source folders are added between backups. So our calculation would still be wrong in many cases.

I'll still mark this as "idea" in case someone else has a better solution.

**Comment by real-yfprojects** [comment]
> Agreed on that. While Borg doesn't have this info

The json object returned by `borg create` and `borg info` have the attribute `stats.original_size` and `stats.deduplicated_size`. So such columns could be populated.

**Comment by m3nu** [comment]
That would be per-archive, not for the whole repo. Won't it clutter the archive table too much?

**Comment by real-yfprojects** [comment]
> That would be per-archive, not for the whole repo.

For the whole repo as well as for an ongoing backup, we already show original and deduplicated+compressed size. So we could calculate the percentage easily.

> Won't it clutter the archive table too much?

That's a good point. We could have a context menu on the table header that allows selecting which columns are displayed. And we could allow reordering columns. Qt also provides a horizontal scrollbar.


Also this FR is closely related to #497.
